docs/application/native/guides/app-management/widget-viewer.md

```diff
@@ -80,19 +80,19 @@ After displaying a widget, you can manage the widget application's lifecycle:
    Evas_Object *widget = widget_viewer_evas_add_widget(win, "widget_id", (const char *)raw, 0.0);
    ```
 
-2. Resume the instance of a widget application using `widget_viewer_evas_resume_widget()`. The parameter of `widget_viewer_evas_resume_widget()` is a widget object created by `widget_viewer_evas_add_widget()`:
+2. Resume the instance of widget application using `widget_viewer_evas_resume_widget()` with the widget object as the parameter.:
 
    ```
    int ret = widget_viewer_evas_resume_widget(widget);
    ```
 
-3. Pause an instance of a widget application instance using `widget_viewer_evas_pause_widget()`. The parameter of `widget_viewer_evas_pause_widget()` is a widget object created by `widget_viewer_evas_add_widget()`:
+3. Pause the instance of widget application using `widget_viewer_evas_pause_widget()` with the widget object as the parameter.:
 
    ```
    int ret = widget_viewer_evas_pause_widget(widget);
    ```
 
-4. Resume or pause all the widget instances displayed by the widget viewer application using `widget_viewer_evas_notify_resumed_status_of_viewer()` and `widget_viewer_evas_notify_paused_status_of_viewer()`:
+4. Resume or Pause all the widget instances displayed by the widget viewer application using `widget_viewer_evas_notify_resumed_status_of_viewer()` and `widget_viewer_evas_notify_paused_status_of_viewer()`.:
```

Comment:
To maintain consistency, please change "widget viewer" to "Widget Viewer" and also change "Pause" to "pause".